Website

jhharwood88 opened the pull request:
Added grid info for body as well as list elements, fixed certain image locations and padding errors once border for page was set.

Commit:
added img folder

Commit:
got the text on the list

Commit:
made grid changes, added button, fixed image locations.
Made the needed grid changes for the checkbox list to fit within the image specified, as well as centering bottom text and adding the share recipie button. centered the doc as well as fixed postions within the doc for given buttos, so it would stop moving around when i changed the body padding.

Commit:
forgot to remove colum1 backgroundimg

Commit:
fixed list layout

Commit:
OOCSS modifications
changed the information paragraph into a single line, rather than 3 seperate ojects. Changed the p tags to div to remove the p. prefix in the css, removed some unnessisary objects when standardizing the layout.

Commit:
changed html, added style sheets
changed html to reflect changes in style sheets, as well as adding a grid layout and putting objects in the grid.

Commit:
made dexters revisions

litteredwitherrors commented on cssporjectredo.html:
```diff
@@ -0,0 +1,86 @@
+      </div>
+      <h1>Delicious</h1>
+      <h2>THE BEST FOOD BLOG ON THE WEB.</h2>
+    <div class="topdirectbar">
```
Revise indentation lines 20-27

litteredwitherrors commented on cssporjectredo.html:
```diff
@@ -0,0 +1,86 @@
+    <link rel="stylesheet" href= "module.css">  
+  </head>
+  <body>
+    <section class ="container_16 .grid_1">
```
`container_16` should only be used for a main containing container.

litteredwitherrors commented on cssporjectredo.html:
```diff
@@ -0,0 +1,86 @@
+    <img src="IMG/insta-icon.png" href="insta-icon">
+    <img src="IMG/flic-icon.png" href="flic-icon">
+    <img src="IMG/pint-icon.png" href="pint-icon">
+    <img src="IMG/mail-icon.png" href="mail-icon"></br>
```
Do not use the br tag. Use css to achieve that effect.

litteredwitherrors commented on cssporjectredo.html:
```diff
@@ -0,0 +1,86 @@
+        </section>
+         <section class="container_16 .grid_5">
+         <div class="borderslash">
+    <img src="IMG/borderslash.png" href="borderslash">
```
This would be better applied as a background image.

Commit:
dexters corrections
revised lines 20-27 indentation to represent proper nesting
moved container 16 onto the body element
removed all br tags from html
have not been able to get borderslash to show up in bg can see in console

dlesaca commented on cssporjectredo.html:
```diff
@@ -0,0 +1,80 @@
+          </ul>
+        </div>
+    </section>
+         <div class="borderslash"></div>
```
This is indented too much

litteredwitherrors replied:
Revise indentation from 64-80

litteredwitherrors commented on cssproject.html:
```diff
@@ -3,15 +3,19 @@
-		<link rel="stylesheet" href=normalize.css>
+		<link rel="stylesheet" href= "normalize.css">
+		<link href='http://fonts.googleapis.com/css?family=Yellowtail' rel='stylesheet' type='text/css'>
 			<style>
```
Move all styles to a separate stylesheet, not in the HTML.

litteredwitherrors commented on cssporjectredo.html:
```diff
@@ -0,0 +1,80 @@
+      </div>
+      <h1>Delicious</h1>
+      <h2>THE BEST FOOD BLOG ON THE WEB.</h2>
+        <div class="topdirectbar">
```
This is indented one too many

litteredwitherrors commented on cssporjectredo.html:
```diff
@@ -0,0 +1,80 @@
+          <img src="IMG/pint-icon.png" href="pint-icon">
+          <img src="IMG/mail-icon.png" href="mail-icon">
+        </div>
+    <section>
```
Revise indentation 29-64

litteredwitherrors commented on style.css:
```diff
@@ -0,0 +1,99 @@
+          bottom: 30px;
+          right: 55px;
+        }
+        .printico{
```
Why did you choose to do it this way? This is very similar to the header icons issue.
